### ISSUE TITLE
Change the default commands on top of body pane to include better ones, such as "Leo-settings", "hoist", etc.

### ISSUE THREAD

**Issue by edreamleo** [boltex/leointeg#163]
1. It's harder to find the settings icon after opening a .leo file.  The documents pane does have a settings icon, but the icon isn't visible until I expand the pane. I suggest adding the (leoInteg) settings icon next to the other icons in the outline pane.

2. There is a quirk with the command pallet:

If I type "Leo s" I see "Leo: Open Settings" and everything is good.

But if I type "Leoin" I see 9 completions, but none of them is "Leo: Open Settings".

So this is confusing. Typing "Leoin" should allow me to see everything that typing "Leo" finds. Or so this newbie thinks.

**Comment by boltex** [comment]
I'm going to re-open the issue that talked about letting the user decide which icon to have on top of body pane, leo outline and leoDocuments. (Because as of now I've only let a small subset of commands for the body pane only) It would be nice indeed to have the 'Leo settings' and 'Leo Welcome' commands available in the UI even after having connected to the server and having Leo files opened. (_even more so that I intend to have 'tabs' with more content in there with general Leo documentation for it's various main topics, as was available in Leo_)

About the Command palette pattern matching with command names: I realised that vscode command palette is NOT like sublime text's Command palette, which offers 'Lazy searching' (or maybe it has to be enabled in vscode - i'm not sure) so yeah: part of the text entered has to match the beginning,or part of the beginning of a word past the extension prefix (Leo).

Also, the philosophy behind the naming is that Leointeg exposes 'Leo' and its commands and and behavior, not 'Leointeg' ... which is just the name of the extension.

So for example, "leo w" should offer the Leo:Welcome command. 

(Sublime text was much more forgiving when trying to match a command to an entered string and would have matched what you suggested - I wish it was so in vscode... like i said maybe theres even a setting for that, i'm still learning vscode, I still mix concepts from sublime to vscode as there both vesry similar hehe! there's like a million settings and commands in there!)

**Comment by boltex** [comment]
Changed my mind - I'll just simply add the 'config' / 'welcome' command to the top body pane menu, (and in the settings webview to show/hide them easily), and change the defaults for which are shown/hidden (vscode limits to 9 total shown so there has to be choices made as to which are shown by default) 

Some of the default ones I've got up there are not optimal in the default visible set (e.g. the clipboard operations are easy to do and memorize on the keyboard, whereas 'refresh-file' can double as an indicator of the body being an external file, etc.) 

So to summarize, adding the 'leo settings' command, and making a cleanup in the visible/hidden default set of command icons on top of the body pane should do the trick :)
